From 25c1570642f8dfff13f2bcc4d137e85f3dac6426 Mon Sep 17 00:00:00 2001
From: "Chai, Chong Yi" <chong.yi.chai@intel.com>
Date: Fri, 1 Jul 2016 09:08:46 +0800
Subject: [PATCH] mosquitto: disable SRV lookup support

Disable SRV lookup support since it requires c-ares and SRV is not 
required by soletta.

Upstream-Status: Inappropriate [disable feature]

Signed-off-by: Chai, Chong Yi <chong.yi.chai@intel.com>
---
 config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.mk b/config.mk
index 1e638b8..14c29bd 100644
--- a/config.mk
+++ b/config.mk
@@ -58,7 +58,7 @@ WITH_MEMORY_TRACKING:=yes
 WITH_SYS_TREE:=yes
 
 # Build with SRV lookup support.
-WITH_SRV:=yes
+WITH_SRV:=no
 
 # Build using libuuid for clientid generation (Linux only - please report if
 # supported on your platform).
-- 
1.9.1

